src/blocks/venue/render.php

```diff
@@ -26,13 +26,76 @@
 
 // (WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase)
 // phpcs:ignore
-$gatherpress_full_address = $gatherpress_venue_information->fullAddress;
+$gatherpress_full_address = $gatherpress_attributes['venueAddress'];
```

Comment:
Change to `$gatherpress_venue_address` should change everywhere. Like this change for consistency with the other variables.